packages/manager/modules/netapp/src/dashboard/volumes/create/template.html

```diff
@@ -0,0 +1,71 @@
+<oui-modal
+    heading="{{:: 'netapp_volume_create_title' | translate }}"
+    loading="$ctrl.isCreating"
+    primary-action="$ctrl.onCreateVolume()"
+    primary-label="{{:: 'netapp_volume_create_title' | translate }}"
+    primary-disabled="$ctrl.createVolumeForm.$invalid"
+    secondary-action="$ctrl.goBack()"
+    secondary-label="{{:: 'netapp_volume_create_cancel' | translate }}"
+    on-dismiss="$ctrl.goBack()"
+>
+    <form action="index.html" novalidate name="$ctrl.createVolumeForm">
+        <oui-field
+            label="{{:: 'netapp_volume_create_name' | translate }}"
+            size="m"
+        >
+            <input
+                class="oui-input"
+                type="text"
+                id="volumeName"
+                name="volumeName"
+                data-ng-model="$ctrl.newVolume.name"
+            />
+        </oui-field>
+        <oui-field
+            label="{{:: 'netapp_volume_create_description' | translate }}"
+            size="m"
+        >
+            <input
+                class="oui-input"
+                type="text"
+                id="volumeDescription"
+                name="volumeDescription"
+                data-ng-model="$ctrl.newVolume.description"
+            />
+        </oui-field>
+        <oui-field
+            label="{{:: 'netapp_volume_create_protocol' | translate }}"
+            size="m"
+        >
+            <oui-select
+                disabled="true"
```

Comment:
Only one option available for this batch, and so select is displayed but read-only

Reply:
```suggestion
                disabled="{{$ctrl.protocolList.length === 1 }}"
```
? :)